### DISCUSSION TITLE
PCIe PCB routing issues might affect all PCIe measurements

### DISCUSSION THREAD

**Discussion by stekgreif** [geerlingguy/raspberry-pi-pcie-devices#659] [General]
I'm working on a board using CM4 and PCIe. I used the CM4IO board to get started but found some issues with the PCB routing, e.g. 90ohms instead of 100ohm, no TX-AC-Cs, differential layout could be improved, ...
It might me worth to compare the CM4IO board with other carrier boards. I would expect different data rates.

**Comment by geerlingguy** [comment]
I would consider asking about this on the Pi Forums, as more engineers hang around in there, and could maybe help shed light on this. This forum is more about testing devices, and less about fixing more general bugs with Pi hardware like the traces on the CM4 IO Board. I'll convert this issue to a discussion :)
